src/rpc/methods/net.rs

```diff
@@ -236,6 +236,28 @@ impl RpcMethod<0> for NetVersion {
     }
 }
 
+pub enum NetProtectAdd {}
+impl RpcMethod<1> for NetProtectAdd {
+    const NAME: &'static str = "Filecoin.NetProtectAdd";
+    const PARAM_NAMES: [&'static str; 1] = ["acl"];
+    const API_VERSION: ApiVersion = ApiVersion::V1;
+    const PERMISSION: Permission = Permission::Admin;
+
+    type Params = (String,);
+    type Ok = ();
+
+    // This is a no-op due to the fact that rust `lib-p2p` implementation is very different to that
+    // in go. However it would be nice to investigate connection limiting options in rust.
+    // See: <https://github.com/ChainSafe/forest/issues/4355>.
+    async fn handle(
+        _: Ctx<impl Blockstore>,
+        (peer_id,): Self::Params,
+    ) -> Result<Self::Ok, ServerError> {
+        let _ = PeerId::from_str(&peer_id)?;
```

Comment:
Can we do nothing?

Reply:
I don't understand. We need to make sure that `PeerId` is valid. And this is the same way it is done for other endpoints that parse `PeerId`.

Reply:
I mean, this method is a no-op, so adding a check may bring incorrect assumptions that we do _something_ with this peer. It's not a blocker.

Reply:
It has to line up with Lotus impl and that will throw an error when the peer_id does not parse.